packages/assets-controllers/src/NftController.ts

```diff
@@ -222,41 +218,235 @@ const controllerName = 'NftController';
 /**
  * The external actions available to the {@link NftController}.
  */
-type AllowedActions =
+export type AllowedActions =
   | AddApprovalRequest
   | NetworkControllerGetNetworkClientByIdAction;
 
+export type AllowEdEvents =
```

Comment:
Typo. All other instances of `AllowEdEvents` need to be fixed as well.

```suggestion
export type AllowedEvents =
```